app/models/pubmed_source_record.rb

```diff
@@ -68,6 +68,19 @@ def self.get_and_store_records_from_pubmed(pmids)
     PubmedSourceRecord.import source_records
   end
 
+  # Retrieve this pubmed record from PubMed and update
+  # is_active, source_data and the source_fingerprint fields.
+  # Used to update the pubmed source record on our end (similar to .sciencewire_update)
+  # @return [Boolean] the return value from update_attributes!
+  def pubmed_update
+    pubmed_source_xml = PubmedClient.new.fetch_records_for_pmid_list pmid
+    pub_doc = Nokogiri::XML(pubmed_source_xml).xpath('//PubmedArticle')[0]
+    attrs = {}
+    attrs[:source_data] = pub_doc.to_xml
+    attrs[:source_fingerprint] = Digest::SHA2.hexdigest(pub_doc)
+    update_attributes! attrs
+  end
```

Comment:
* a single value should work, i tested it
* that is a good idea, I'll make a ticket for it, but outside the scope of this rather limited change for which I just want to run a specific set of author IDs